src/Components/WhyWeDidItSection/WhyWeDidItSection.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './WhyWeDidItSection.css';
+import { Typography } from 'antd';
+
+const { Title, Paragraph } = Typography;
+
+function WhyWeDidItSection({ imgSrc, title, text }) {
```

Comment:
if the text, title and image doesn't change you don't need them as props just hardcode them in the component itself

Reply:
we'll call this component 3 times and each time different props similar to this section in this website
(Im tryna attach a screen-shot to show u what i meant but it's not uploading dunno why, i'll send it on Slack)